src/dist/replication/meta_server/meta_bulk_load_service.h

```diff
@@ -43,6 +43,16 @@ struct bulk_load_info
     DEFINE_JSON_SERIALIZATION(app_id, app_name, partition_count)
 };
 
+template <typename T>
+inline void erase_map_elem_by_id(int32_t app_id, std::unordered_map<gpid, T> &mymap)
+{
+    for (auto iter = mymap.begin(); iter != mymap.end();) {
+        if (iter->first.get_app_id() == app_id) {
+            mymap.erase(iter++);
```

Comment:
Missing `break`?

Reply:
The structure of mymap is <gpid,T>, this function is to remove all items whose `gpid.app_id == app_id`, so this function doesn't need break if remove one item.